src/helpers.js

```diff
@@ -145,6 +145,12 @@ export function convertToUnitVerbose(input, inputUnit, outputUnit) {
     return result;
 }
 
+export function formatWithBestUnit(input, inputUnit) {
+    const best = convertToBestUnit(input, inputUnit || units.B);
+    const decimals = best.value >= 100 ? 0 : best.value >= 10 ? 1 : 2;
+    return cockpit.format("$0 $1", best.value.toFixed(decimals), best.unit);
+}
```

Comment:
"meh". The existing converToBestUnit() should die, die, die: it is base-2 (and thus inconsistent with all other cockpit pages which are base-10), and just duplicates the logic of `cockpit.format_number()`. I see zero reason to not just use the latter for the dialog, and then we should make a separate PR to get rid of all these helpers in machines. You don't even use `inputUnit` from anywhere.

Reply:
Yeah... I was sticking with existing practices, but I am happy to use the official functions. I didn't pay attention to the recent overhaul... would I just use `cockpit.format_bytes(input)` here?

Reply:
Yes, that automatically selects the right unit, in base-10. That should be fine for this dialog.